tools/ci-build.sh

```diff
@@ -67,6 +67,31 @@ cd /p4c
 backends/ebpf/build_libbpf
 cd -
 
+function install_ptf_ebpf_test_deps() (
+  export P4C_PTF_PACKAGES="gcc-multilib \
+                           python3-six \
+                           libjansson-dev \
+                           linux-tools-`uname -r`"
+  # Package "linux-tools-generic-hwe-20.04" is not required because
+  # we test under current kernel, not the newest one
+  apt-get install -y --no-install-recommends ${P4C_PTF_PACKAGES}
+
+  git clone --recursive https://github.com/P4-Research/psabpf.git /tmp/psabpf
+  cd /tmp/psabpf
+  # FIXME: psabpf is under heavy development, later use git tags when it will be ready to use
+  git reset --hard 8fc9687
```

Comment:
Would it be possible to create a "stable" branch in the psabpf repository and use this branch to test p4c?

This would make it a lot easier to run the tests locally.